src/main/java/com/thevoxelbox/voxelsniper/voxelsniper/Environment.java

```diff
@@ -0,0 +1,6 @@
+package com.thevoxelbox.voxelsniper.voxelsniper;
+
+public enum Environment {
+    BUKKIT,
+    FORGE,
+}
```

Comment:
Do we (still) need this? Doesn't seem to be used outside of tests..

Reply:
Not at this moment, but this could be useful in the future if we get version specific features

Reply:
Good point